File: mulTables.java
import java.util.Scanner;

public class Main {	
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int table;
		table = in.nextInt();
		
		for(int i = 0; i <= 10; i++){
			int result = table * i;
			System.out.println(table + " X " + i + " = " + result);
		}
	}
}

File: mulTables.cpp
#include <bits/stdc++.h>
#include <iostream>

using namespace std;
int main(){
	int table;
	cin >> table;

	for(int i = 0; i <= 10; i++){
		int result = table * i;
		cout << table << " X " << i << " = " << result << endl;
	}

	return 0;
}
